app/utils/FieldValidator.scala

```diff
@@ -87,8 +87,10 @@ object FieldValidator {
 
   val startsWithCapitalLetter: String => Boolean = (input: String) => input.headOption.exists(_.isUpper)
 
-  val isContainedIn: Iterable[String] => String => Boolean =
-    (iterable: Iterable[String]) => (input: String) => iterable.exists(_ == input)
+  def isContainedIn(iterable: Iterable[String]): String => Boolean = (input: String) => iterable.exists(_ == input)
+
+  def isContainedIn[T](iterable: Iterable[T], stringify: T => String): String => Boolean =
+    (input: String) => iterable.map(stringify).exists(_ == input)
```

Comment:
applying `stringify` on exist allow to kick short circuit on exist - could not copy whole collection if element would be in beginning of collection

Reply:
I dont think we need to be optimizing to the nanosecond ;)